docs/advanced/advanced.md

```diff
@@ -11,40 +11,25 @@ import TabItem from '@theme/TabItem';
 
 This section contains advanced information describing the different ways you can run and manage K3s, as well as steps necessary to prepare the host OS for K3s use.
 
-## Certificate Rotation
+## Certificate Management
 
-### Automatic rotation
-By default, certificates in K3s expire in 12 months.
+### Certificate Authority Certificates
 
-If the certificates are expired or have fewer than 90 days remaining before they expire, the certificates are rotated when K3s is restarted.
+K3s generates self-signed Certificate Authority (CA) Certificates during startup of the first server node. These CA certificates are valid for 10 years, and are not automatically renewed.
 
-### Manual rotation
+For information on using custom CA certificates, or renewing the self-signed CA certificates, see the [`k3s certificate rotate-ca`](../reference/cli/certificate.md#rotate-ca) CLI tool documentation.
```

Comment:
```suggestion
For information on using custom CA certificates, or renewing the self-signed CA certificates, see the [`k3s certificate rotate-ca` documentation](../reference/cli/certificate.md#rotate-ca).
```

I'm not in love with embedding the link within a code-block, because it hides the blue link color. So expanding the link to include "documentation" helps the user.

Reply:
that's fair. I can make sure that any where I'm linking with monospace there are also normal words included in the link.